### PR TITLE
Fug bixes

### PR DESCRIPTION
**Pull request by kevinshieh**
random
issue #121 
issue #126 
issue #124 
issue #135

### CODE REVIEW

File: app/views/application/_header.html.erb [hunk 1]
```diff
@@ -1,12 +1,5 @@
 <div class="header">
   <%= link_to "Apply", apply_path, class: "header-tab header-button" %>
-
-  <% if !current_user %>
-    <!-- Temporary log in link -->
-    <%= link_to "Log in", new_admin_session_path, class: "header-tab" %>
-  <% else %>
-    <%= link_to "Log out", destroy_user_session_path, class: "header-tab", method: :delete %>
-  <% end %>
```
Comment (SamLau95) [inline app/views/application/_header.html.erb]: Let's not remove the logout link because otherwise there's no other way to log out haha!

Comment (kevinshieh) [inline app/views/application/_header.html.erb]: orite. i assumed you would just hit the route

Comment (SamLau95) [inline app/views/application/_header.html.erb]: Yup haha. Also cause the HTTP verb has to be delete so typing in the URL in the browser doesn't work haha. This logout link logs out for all types of users, including students and NPOs, so let's keep it.

**Comment by SamLau95** [comment]
Also, if you write

Fixes {some issue number}
or
Closes {some issue number}

In your pull request it'll close the issues when it gets merged!

**Pull request merged**